app/controllers/works_controller.rb

```diff
@@ -691,17 +691,25 @@ def update_multiple
     @user = current_user
     @works = Work.joins(pseuds: :user).where('users.id = ?', @user.id).where(id: params[:work_ids]).readonly(false)
     @errors = []
-    # to avoid overwriting, we entirely trash any blank fields and also any unchecked checkboxes
-    updated_work_params = work_params.reject { |_key, value| value.blank? || value == '0' }
 
-    # manually allow switching of anon/moderated comments
+    # To avoid overwriting, we entirely trash any blank fields.
+    updated_work_params = work_params.reject { |_key, value| value.blank? }
+
+    # We used to reject any parameters with value "0", so on the old edit
+    # multiple form there were special values in place of the "0". We keep
+    # checking those values for backwards-compatibility with the old form.
```

Comment:
I don't think anyone's going to have an Edit Multiple Works form open very long, so it can be removed soon. But I want it to work okay if someone loads the form right before the deploy, and submits it right after, so I left some legacy code in. I'll add an explicit TODO.